Gopkg.lock

```diff
@@ -120,6 +120,12 @@
   packages = ["unix","windows"]
   revision = "fb4cac33e3196ff7f507ab9b2d2a44b0142f5b5a"
 
+[[projects]]
+  branch = "v2"
+  name = "gopkg.in/yaml.v2"
+  packages = ["."]
+  revision = "cd8b52f8269e0feb286dfeef29f8fe4d5b397e0b"
+
```

Comment:
"yes", but:

- although json is easier to read than XML, it's still not super-easy to read.
- we are admittedly using toml for our config file (I actually don't know where that requirement came from), but I personally find that its embedded tables somewhat odd.

YAML is imho the most human-readable + machine-readable format around atm.